tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/fly.ts

```diff
@@ -0,0 +1,41 @@
+import { createLanguagePerk, Species } from "./base";
+
+const Fly: Species = {
+  description: "With no official documentation or knowledge of the origin of \
+    this species, they remain a mystery to most. Any and all rumours among \
+    Nanotrasen staff regarding flypeople are often quickly silenced by high \
+    ranking staff or officials.",
+  features: {
+    good: [{
+      icon: "grin-tongue",
+      name: "Uncanny Digestive System",
+      description: "Flypeople regurgitate their stomach contents and drink it \
+        off the floor to eat and drink with little care for taste, favoring \
+        gross foods.",
+    }, createLanguagePerk("Buzzwords")],
+    neutral: [],
+    bad: [{
+      icon: "fist-raised",
+      name: "Insectoid Biology",
+      description: "Fly swatters will deal significantly higher ammounts of \
```

Comment:
```suggestion
      description: "Fly swatters will deal significantly higher amounts of \
```

Though personally, I chose to avoid specific interactions like this from other species. You don't see this on moths, for instance, even though they have a similar weakness (though not as bad)

Reply:
Yeah I wasn't sure on this one too, but having flyswatters do 30x damage is too big of a weakness to leave out, since someone could pick the race up not knowing this and die to it.